lib/ansible/modules/windows/win_file.py

```diff
@@ -47,19 +47,20 @@
         the C(path) will be deleted and set as a new junction point.
       - If state=C(hard) and C(path) exists and is not a hard link of C(src),
         the C(path) will be deleted and set as a new hard link.
-    choices: [true, false]
+    type: bool
+    default: no
```

Comment:
Yup, so people really should be adding no or yes without quotes. We want the "no" and "yes" be listed in the documentation as-is, and not as False or True. (Which no would translate to in YAML, unless it is quoted).